mypy/build.py

```diff
@@ -120,6 +120,7 @@ def build(sources: List[BuildSource],
           pyversion: Tuple[int, int] = defaults.PYTHON3_VERSION,
           custom_typing_module: str = None,
           implicit_any: bool = False,
+          fast_parser: bool = False,
```

Comment:
I would like there to be fewer boolean flags, not more. The original pattern is to define a string for each option (e.g. VERBOSE) in build and to add that string to the flags argument when the option is given. Then checking whether a flag is set can be done by asking whether e.g. `VERBOSE in flags`. This way we can support many flags without a proliferation of boolean arguments. You might want to switch to this pattern for implicit_any too.